easybuild/framework/easyblock.py

```diff
@@ -2435,8 +2435,12 @@ def build_and_install_one(ecdict, init_env):
             # collect build stats
             _log.info("Collecting build stats...")
 
-            buildstats = get_build_stats(app, start_time, build_option('command_line'))
-            _log.info("Build stats: %s" % buildstats)
+            if build_option('module_only'):
```

Comment:
@boegel As far as I can remember, I never had the need to use `--skip`.  That is, I first need to look up what it is actually doing ;)  Do you have any particular use case in mind that would cause problems?